packages/core/src/ui/InlineToolbar/index.ts

```diff
@@ -129,7 +138,7 @@ export class InlineToolbar {
       inlineElementButton.innerHTML = toolName;
 
       inlineElementButton.addEventListener('click', (_event) => {
-        this.apply(toolName as InlineToolName);
+        this.renderToolActions(createInlineToolName(toolName));
```

Comment:
create `isActionsAvailable` method in facade and call 2 different method: one for applying and one for showing actions